localfs/system_windows.go

```diff
@@ -0,0 +1,40 @@
+package localfs
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/sys/windows"
+)
+
+func localToQid(path string, info os.FileInfo) uint64 {
+	pathPtr, err := windows.UTF16PtrFromString(path)
+	if err != nil {
+		fmt.Println("hit 1: ", path)
+		panic(err) //FIXME: port wort
+	}
+
+	var (
+		access     uint32 = 0 // we only need metadata
+		sharemode  uint32 = 0
+		createmode uint32 = windows.OPEN_EXISTING
+		attribute  uint32 = windows.FILE_ATTRIBUTE_NORMAL
+	)
+	if info.IsDir() {
+		attribute = windows.FILE_FLAG_BACKUP_SEMANTICS
+	}
+	fd, err := windows.CreateFile(pathPtr, access, sharemode, nil, createmode, attribute, 0)
+	if err != nil {
+		fmt.Println("hit 2: ", pathPtr, "\n", path, "\n", windows.GetLastError())
```

Comment:
The prints should be removed at the very least but I was reluctant to change the function signature (appending an error to the return).
If Go was able to retrieve a stat for the file already, it's highly unlikely we'd be unable to access it again here, using the system native methods.
That being said, a panic is a panic, so maybe it would be best to change ` func localToQid(path string, info os.FileInfo) uint64` to ` func localToQid(path string, info os.FileInfo) (uint64, error)`